website/content/partials/telemetry-metrics/vault/raft/delete_prefix.mdx

```diff
@@ -0,0 +1,5 @@
+### vault.raft.delete_prefix ((#vault-raft-delete_prefix))
```

Comment:
Again, this isn't a valid metric.  There's a raft_storage.fsm.delete_prefix, which I believe we do use, but its meaning is complicated and hard to explain without going into a lot of detail on raft chunking.

Reply:
I deleted the partial and removed it from the relevant index pages. Let me know if we should add an entry for raft_storage.fsm.delete_prefix (it doesn't exist in the current doc)